docs/submitting_data.rst

```diff
@@ -206,7 +208,8 @@ data structures to send, do something like this.
     #
     # data structure validation is automatically performed here, if validation
     # fails a TreeherderClientError is raised
-    client.post_collection('mozilla-central', 'oauth_key', 'oauth_secret', trc)
+    auth = TreeherderAuth('oauth_key', 'oauth_secret', 'mozilla-central')
+    client.post_collection('mozilla-central', trc, auth=auth)
```

Comment:
I'd probably set up auth when we construct the client, as it sounds like that will be the more common use case